plugins/jabra/fu-plugin-jabra.c

```diff
@@ -33,7 +33,7 @@ fu_plugin_update_cleanup (FuPlugin *plugin,
 
 	/* check for a property on the *dfu* FuDevice, which is also why we
 	 * can't just rely on using FuDevice->cleanup() */
-	if (!fu_device_has_custom_flag (device, "attach-extra-reset"))
+	if (!fu_device_has_vendor_id (device, "USB:0x0B0E"))
```

Comment:
Perhaps should this become an internal plugin flag since it needs to go across two plugins?

Reply:
Yes, this did feel a bit of hack. Move it to a ATTACH_EXTRA_RESET flag perhaps?

Reply:
Yup

Reply:
Fixed, indirectly.